src/main/java/geometry/Shape.java

```diff
@@ -0,0 +1,25 @@
+package geometry;
+
+public interface Shape {
+
+    static void main(String[] args) {
```

Comment:
please move main() method from interface, create another class or just paste method's content to some new `@Test` method in TestShape class (you can use test methods not only to test something with assertions, but to run some code as well)

Reply:
done